flink-examples/flink-examples-build-helper/flink-examples-streaming-state-machine/pom.xml

```diff
@@ -71,15 +71,16 @@ under the License.
 							</transformers>
 							<artifactSet>
 								<includes>
-									<include>org.apache.flink:flink-connector-kafka*</include>
-									<include>org.apache.flink:flink-examples-streaming*</include>
+									<include>org.apache.flink:flink-connector-datagen</include>
+									<include>org.apache.flink:flink-connector-kafka</include>
+									<include>org.apache.flink:flink-examples-streaming</include>
 									<include>org.apache.kafka:*</include>
 								</includes>
 							</artifactSet>
 
 							<filters>
 								<filter>
-									<artifact>org.apache.flink:flink-examples-streaming_*</artifact>
+									<artifact>org.apache.flink:flink-examples-streaming</artifact>
```

Comment:
It's better to split these changes (fix wrong filtering and remove Scala version) into a hotfix commit at the top of this PR.

Reply:
Thanks for the comment, I'll separate the commits.

Reply:
Usually, we add hotfix before other changes.

Reply:
Sure, I'll rearrange the commits order and force-push again. Thanks for pointing that.